specs/network_details_for_hardware_inventory.adoc

```diff
@@ -0,0 +1,112 @@
+= Specification for adding network details in node hardware inventory
+
+Add network details also as a part of node hardware inventory
+
+== Problem description
+
+Hardware inventory details are used to show exact status of the particular
+node. Currently network details are missing in node hardware inventory.
+we have to add network details like IPv4, IPv6, netmask as a part of node
+hardware inventory.
+
+== Use Cases
+
+Tendrl should show the current status of active network interface.
+
+== Proposed change
+
+* Modify the manager module to find active network interfaces details except
+  lo, virbr0, virbr0-nic and update those details into node_inventory list.
+* Create a new class in persistence module to keep network details.
+* Populate the object using node_inventory list and persist the object in etcd.
+
+== Alternatives
+
+* Netifaces package is used to find active network interface in particular
+  node.
+* Add netifaces package as dependency.
+
+== Data model impact
+
+Add network details in etcd with key nodes/node_id/Network/interface_name
+
+== Impacted modules
+
+=== Tendrl API impact:
+
+Api to get network details v2/keys/nodes/node_id/Network
+
+=== Tendrl/common impact
+
+None
+
+=== Tendrl/node_agent impact
+
+Modify persistence and manager modules to find and store active network
+interface details.
+
```

Comment:
What about the definitions? Do we have an object defined for network? 
Also any new atoms needs to be added?